mcs_benchmark_data/pipelines/mcscript/trian_mcscript_submission_file_names.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+from dataclasses_json import LetterCase, dataclass_json
+
+from mcs_benchmark_data._benchmark_file_names import (
+    _BenchmarkFileNames,
+)
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass(frozen=True)
+class TrianMCScriptSubmissionFileNames(_BenchmarkFileNames):

```

Comment:
You don't really need a subclass if you're just going to instantiate it in one place. That was the purpose of having a a few "patterns" of _BenchmarkFileNames.